src/lib/ai/mental-llama/PythonBridge.ts

```
@@ -63,96 +283,339 @@ export class MentalLLaMAPythonBridge {
+    try {
+      // Run the analyzer
+      const output = await this.executeCommandWithOutput(pythonExe, [
+        analyzerScript,
+      ])
+
+      // Parse the results
+      const results = JSON.parse(output)
+
+      // Clean up temporary files
+      try {
+        const fs = require('fs')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)